Add hyperlinks to bitmap and bloom filter files (#318)
valkey-io/valkey-doc: commands/bitcount.md

It is possible to specify the counting operation only in an interval passing the
additional arguments _start_ and _end_.

> Like for the [GETRANGE](getrange.md) command, start and end can contain negative values in

Reviewer: ```suggestion
Similar to the [GETRANGE](getrange.md) command, `start` and `end` can contain negative values in
```

for instance it is possible to determine what users are good targets of beta
features.

> Using the [SETBIT](setbit.md) command this is trivial to accomplish, identifying every day

Reviewer: ```suggestion
Using the [SETBIT](setbit.md) command, this is trivial to accomplish, identifying every day
```
